packages/manager/modules/pci/src/projects/project/storages/databases/database/users/add/add.controller.js

```diff
@@ -154,10 +154,20 @@ export default class AddUserCtrl {
     return user;
   }
 
+  checkUsernameExist(username) {
+    this.usernameCheck = username;
```

Comment:
**nitpick:** Looks `this.usernameCheck` is not used anywhere. Can we avoid creating a key on `this`?

Reply:
I stored the value to read it inside `data-translate-values`. It permits to stored the previous input value of the user.